cli-wrap/src/main/resources/cli/da_cli_script.py

```diff
@@ -10,12 +10,16 @@
 
 with open('config.json') as config_file:    
     config = json.load(config_file)
-da_server = os.getenv('DA_SERVER', config["daServer"])
+da_server_raw = os.getenv('DA_SERVER', config["daServer"])
 
-if (da_server == ""):
-    print("Please configure DA server by command $ export DA_SERVER=your.adress/rest/v-1")
+if (da_server_raw == ""):
+    print("Please configure DA server by command $ export DA_SERVER=your.adress")
     print("or by filling address in configuration file (config.json).")
     exit()
+    
+da_server = da_server_raw + "/da/rest/v-1"
+da_server_ws = da_server_raw + "/da/ws"
```

Comment:
I realized that REST uses HTTP protocol and websockets uses WS protocol. The url in config should be protocol less and you should add the protocol here.